WebDevStudios/constant-contact-forms
# CC-162: Replace Frontend Form Element IDs with Classes

Ticket: [CC-162](https://webdevstudios.atlassian.net/browse/CC-162)

### Description ###

Updates frontend form elements to use class instead of ID when ID is not unique.
Updates styles and JS to reference via class instead of ID.
Fixes submit button targeting for recaptcha v2.
Fixes intermittent issue with `ctct-instance` field triggering error in API request.

### Steps to verify the solution ###

1. Add multiple forms on a page, with no recaptcha for first form and recaptcha v2 for the rest.
2. On recaptcha validation, ensure only corresponding submit button is enabled (and disabled when validation times out).
3. Confirm form elements/inputs are using class instead of ID where necessary.
4. Confirm contact is added to CC account.
